src/uint256.h

```diff
@@ -53,15 +53,27 @@ class base_blob
         std::fill(m_data.begin(), m_data.end(), 0);
     }
 
+    /** Lexicographic ordering
+     * @note Does NOT match the ordering on the corresponding \ref
```

Comment:
this is really messy, I wonder if we have a chance to untangle any of this - or whether documenting it is our only possible solution